packages/terra-consumer-nav/src/components/user-profile/UserProfile.jsx

```diff
@@ -50,26 +47,30 @@ const propTypes = {
 };
 
 const defaultProps = {
-  avatar: <IconPerson />,
   profileLinks: [],
-  isSignIn: false,
   id: 'terra-conumser-nav-profile-button',
 };
 
 const UserProfile = ({
-  userName, avatar, id, signoutUrl, signinUrl, isSignIn, profileLinks, handleClick, intl, ...customProps
+  userName, avatar, id, signoutUrl, signinUrl, profileLinks, handleClick, intl, ...customProps
 }) => {
   let profileContent;
-  if (isSignIn) {
-    profileContent = (
-      <button className={cx('popup-button')} href={signinUrl}>
-        <Arrange
-          fitStart={<div className={cx('avatar')}>{avatar}</div>}
-          fill={<FormattedMessage id="Terra.Consumer.UserProfile.signin" />}
-          align="center"
-        />
-      </button>
-    );
+  const avatarIcon = <div className={cx('avatar')}>{avatar || <IconPerson />}</div>;
+
+  const setStaticProfile = (url, translationId) => (
```

Comment:
what are we setting? aren't we just returning a profile arrange?

Reply:
I just wrote this function so i didn't have to duplicate code for the sign-in/sign-out profile link, since all that changed between the two is the text and the link

Reply:
I'm just being picky about the name. Drying up the code is fine.

Reply:
will come up with a better name haha